src/Util/ClassSourceManipulator.php

```diff
@@ -1113,6 +1113,8 @@ private function getEntityTypeHint($doctrineType)
                 return '\\'.\DateInterval::class;
 
             case 'object':
+                return \PHP_VERSION_ID < 70200 ? null : 'object';
```

Comment:
This check should be done in the `PhpCompatUtil` -> https://github.com/symfony/maker-bundle/blob/main/src/Util/PhpCompatUtil.php

Something along the lines of `PhpCompatUtil::canUseObjectReturnType()`

Reply:
@jrushlow Done

Reply:
@jrushlow Since the maker bundle now requires PHP 7.2.5+ I've removed all conditional logic.